app.js

```diff
@@ -62,29 +29,69 @@ var getCookiesPerHour = function(objectName){
   }
 };
 
+CookieStore.prototype.makeTheTable = function(tableToDrawInto){
```

Comment:
Does this make the whole table, or just a part of it? Can you think of a clearer name?